packages/kit/src/views/Market/MarketDetailV2/components/TokenActivityOverview/utils/formatTokenActivityData.ts

```
@@ -0,0 +1,42 @@
+        buys = Number(tokenDetail.buy4hCount) || 0;
+        sells = Number(tokenDetail.sell4hCount) || 0;
+        buyVolume = Number(tokenDetail.volumeBuy4h) || 0;
+        sellVolume = Number(tokenDetail.volumeSell4h) || 0;
+        break;
+      case '24h':
+        buys = Number(tokenDetail.buy24hCount) || 0;
+        sells = Number(tokenDetail.sell24hCount) || 0;
+        buyVolume = Number(tokenDetail.volumeBuy24h) || 0;
+        sellVolume = Number(tokenDetail.volumeSell24h) || 0;
+        break;
+      case '1h':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)